BootloaderCorePkg/Library/PciEnumerationLib/PciEnumerationLib.c

```diff
@@ -460,6 +461,13 @@ CreatePciIoDevice (
     CopyMem (& (PciIoDevice->Pci), Pci, sizeof (PCI_TYPE00));
   }
 
+  PciIoDevice->IsPciExp = FALSE;
+  PciIoDevice->PciExpressCapabilityOffset = 0;
+  if (FeaturePcdGet (PcdAriSupport)) {

```

Comment:
Do we plan to support SR-IOV later on ?

Reply:
Yes, I am working on it in a separate PR.